gordian/repo.py

```diff
@@ -16,7 +14,7 @@
 
 class Repo:
 
-    def __init__(self, repo_name, github_api_url=None, branch=None, github=None, files=None, semver_label=None, target_branch='master', fork=False):
+    def __init__(self, repo_name, github_api_url=None, branch=None, github=None, files=None, semver_label=None, target_branch='master', fork=False, token=None):
```

Comment:
I think we should be consistent, if we allow to pass the token in the constructor we should also allow username and password, so the user can choose between environment variables or passing the bits in the constructor

Reply:
💯 , agreed - I'll get those changes in 😄

Reply:
Updated in: https://github.com/argoproj-labs/gordian/pull/41/commits/181c96669de54d95489c4914854a14f067d65ab3